### PR TITLE
Add linewidth parameter

### PR DESCRIPTION
**Pull request by evanofslack**
#### Reference Issues/PRs
#101

#### What does this implement/fix? Explain your changes.
Implements the linewidth parameter. Includes updates to the `plot` method,  `plot_params_filter` function as well as the various `load` and `save` functions.

### CODE REVIEW

**Review by sadrasabouri — APPROVED** [review]
Good job!
I think it's OK.
@sepandhaghighi please take a look at it.

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
```
Comment (sepandhaghighi) [inline CHANGELOG.md:7]: Please add `### Changed` here

File: samila/params.py [hunk 2]
```diff
@@ -19,6 +19,7 @@
 DEFAULT_COLOR = "black"
 DEFAULT_BACKGROUND_COLOR = "white"
 DEFAULT_ALPHA = 0.1
+DEFAULT_LINEWIDTH = 1.0
```
Comment (sepandhaghighi) [inline samila/params.py:22]: `1.0` or  `1.5`?

Reference: [https://matplotlib.org/stable/api/_as_gen/matplotlib.axes.Axes.scatter.html#matplotlib.axes.Axes.scatter](https://matplotlib.org/stable/api/_as_gen/matplotlib.axes.Axes.scatter.html#matplotlib.axes.Axes.scatter)

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]
@evanofslack Thanks for your effort 💯 
Please take a look at my comments (minor issues)

File: CHANGELOG.md (hunk 1 above)
Comment (evanofslack) [reply CHANGELOG.md:7]: Done!

**Review by sepandhaghighi — APPROVED** [review]
LGTM 🚀

**Pull request merged**